x-pack/plugin/esql/qa/testFixtures/src/main/resources/inlinestats.csv-spec

```diff
@@ -67,29 +67,85 @@ emp_no:integer | avg_worked_seconds:long | gender:keyword | max_avg_worked_secon
          10030 |               394597613 | M              | 394597613
 ;
 
-// TODO allow inline calculation like BY l = SUBSTRING(
 maxOfLongByCalculatedKeyword
-required_capability: inlinestats
+required_capability: inlinestats_v2
 
 // tag::longest-tenured-by-first[]
+FROM employees
+| KEEP emp_no, avg_worked_seconds, last_name
+| INLINESTATS max_avg_worked_seconds = MAX(avg_worked_seconds) BY SUBSTRING(last_name, 0, 1)
+| WHERE max_avg_worked_seconds == avg_worked_seconds
+| SORT last_name ASC
+| LIMIT 5
+// end::longest-tenured-by-first[]
+;
+
+// tag::longest-tenured-by-first-result[]
+emp_no:integer | avg_worked_seconds:long | last_name:keyword | SUBSTRING(last_name, 0, 1):keyword | max_avg_worked_seconds:long
```

Comment:
The output order here is inconsistent: it should be

```
emp_no:integer | avg_worked_seconds:long | last_name:keyword | max_avg_worked_seconds:long | SUBSTRING(last_name, 0, 1):keyword 
```
otherwise `INLINESTATS` with and without expressions place the groups in different places.

Reply:
Ergh, sorry, please disregard; I think I was staring at this for too long, it's actually fine.